lms/djangoapps/instructor/services.py

```diff
@@ -78,13 +76,25 @@ def delete_student_attempt(self, student_identifier, course_id, content_id, requ
             except (StudentModule.DoesNotExist, enrollment.sub_api.SubmissionError):
                 err_msg = (
                     'Error occurred while attempting to reset student attempts for user '
-                    '{student_identifier} for content_id {content_id}.'.format(
-                        student_identifier=student_identifier,
-                        content_id=content_id
-                    )
+                    f'{student_identifier} for content_id {content_id}.'
                 )
                 log.error(err_msg)
 
+    def complete_student_attempt(self, user_identifier, content_id):
+        """
+        Calls the complete_student_attempt_task which
+        Submits all completable xblocks inside of the content_id block to the
+        Completion Service to mark them as complete. One use case of this function is
```

Comment:
The first line of a docstring should be a one-line summary. https://www.python.org/dev/peps/pep-0257/